app/models/donation_chart.rb

```diff
@@ -8,12 +8,25 @@ def self.donation_by_date(candidate)
     donation_date = donation_date.to_s.gsub('"', '')
   end
 
+  def self.donation_issues
+    Issue.all.map {|issue| issue.topic }
+  end
+
+  def self.issue_donation_total
+    Issue.all.map do |issue|
+      issue.candidate_issues.map { |candidate_issue| candidate_issue.donations.
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.